system-monitor@paradoxxx.zero.gmail.com/schemas/org.gnome.shell.extensions.system-monitor.gschema.xml

```diff
@@ -398,6 +398,10 @@
     </key>
     <key name="gpu-used-color" type="s">
       <default>'#00b35b'</default>
+      <summary>Color of program gpu usage in the chart</summary>
```

Comment:
Please use GPU in capital letters.

Reply:
Fixed capitalization of GPU. I noticed there were other places where GPU had other case, so I fixed them all.